plugins/data-management/src/AddTrackWidget/components/ConfirmTrack.tsx

```diff
@@ -98,6 +126,102 @@ function getTrackTypes(pluginManager: PluginManager) {
   return pluginManager.getElementTypesInGroup('track') as { name: string }[]
 }
 
+const TextIndexingConfig = observer(({ model }: { model: AddTrackModel }) => {
+  const classes = useStyles()
+  const [value1, setValue1] = useState('')
+  const [value2, setValue2] = useState('')
+  const [attributes, setAttributes] = useState(['Name', 'ID', 'type'])
+  const [exclude, setExclude] = useState(['CDS', 'exon'])
+  const sections = [
+    {
+      label: 'Indexing attributes',
+      values: attributes,
+    },
+    {
+      label: 'Feature types to exlcude',
+      values: exclude,
+    },
+  ]
+  model.setTextIndexingConf({ attributes, exclude })
```

Comment:
generally it is a little bit of a red flag to have model actions directly inside the rendering of a component. maybe as a useEffect